rocketchat_API/APISections/groups.py

```diff
@@ -106,6 +106,15 @@ def groups_rename(self, room_id, name, **kwargs):
             "groups.rename", roomId=room_id, name=name, kwargs=kwargs
         )
 
+    def groups_set_announcement(self, room_id, announce, **kwargs):
```

Comment:
I'd like to keep it as close to the API naming as possible as long as we don't break Python's style
```suggestion
    def groups_set_announcement(self, room_id, announcement, **kwargs):
```